integrasjon/dokgen-klient/src/main/java/no/nav/foreldrepenger/melding/integrasjon/dokgen/dto/innvilgelsefp/Prosent.java

```diff
@@ -0,0 +1,46 @@
+package no.nav.foreldrepenger.melding.integrasjon.dokgen.dto.innvilgelsefp;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public class Prosent {
+
+    @JsonValue
+    private BigDecimal verdi;
+
+    private Prosent(BigDecimal verdi) {
```

Comment:
Kanskje også gjort et mulig å spesifisere presisjon i ctr?

Reply:
Alle verdiobjekter bør ha en toString()

Reply:
Er alle verdier lovlige, negative og > 100 ? Hvis ikke bør de feilene fanges i ctr

Reply:
Ser ikke behov for å eksponere dette så lenge Prosent bare er en lokal klasse til innvilgelsesbrevet, der man skal ha kun én desimal alle stedene. Litt av gevinsten med klassen er nettopp å sentralisere presisjonshåndteringen på ett sted så det blir som forventet for hele brevet.

Reply:
OK, tenker at slike helt generelle verdiklasser egentlig hører hjemme i felles, men da må de være mer generelle

Reply:
Over 100 er lovlig (og rundes evt. ned til 100 for visningen, da 100% gir egen tekst uten prosent), men kan legge til guard på negativ, da jeg tviler på at det er en gyldig verdi.